shieldlib/src/shield_canvas_draw.ts

```diff
@@ -97,6 +97,16 @@ function ellipse(
   return width;
 }
 
+export function pill(
+  r: ShieldRenderingContext,
+  ctx: CanvasRenderingContext2D,
+  params: ShapeBlankParams,
+  ref?: string
+): number {
+  params.radius = r.shieldSize() / 2;
```

Comment:
This doesn't seem to be working right for 200% zoom:

[![image](https://github.com/ZeLonewolf/openstreetmap-americana/assets/1732117/79172f28-017b-48d7-8b68-a99a6b1c4294)](https://preview.ourmap.us/pr/1069/#map=14.2/38.19356/-81.49928)

Reply:
Fixed in dd2192e. I also added a note explaining the logic at that spot in the code.